ecosystem/indexer-grpc/indexer-grpc-data-service/src/service.rs

```diff
@@ -437,20 +447,12 @@ impl RawData for RawDataServerWrapper {
 
 /// Builds the response for the get transactions request. Partial batch is ok, i.e., a batch with transactions < 1000.
 fn get_transactions_responses_builder(
-    data: Vec<EncodedTransactionWithVersion>,
+    transactions: Vec<Transaction>,
```

Comment:
Can you delete the `log_grpc_step` in this function? The decoding step does not happen in this function anymore.